schema-util/common/src/main/java/io/apicurio/registry/content/dereference/DereferencingNotSupportedException.java

```diff
@@ -0,0 +1,23 @@
+package io.apicurio.registry.content.dereference;
+
+import io.apicurio.registry.types.RegistryException;
+
+public class DereferencingNotSupportedException extends RegistryException {
```

Comment:
What http response error code is this mapped to?

Reply:
None, I forgot to add it to the mapper, thanks.